ikenna-portfolio/portfolio/src/main/java/com/google/sps/data/Metadata.java

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+* The Metadata class stores information that represents how the comment container
+* will look like.
+* 
+*  
+*/
+public class Metadata {
+  protected final int count;
+  protected final int page;
+  protected int maxPages;
+  protected final boolean ascending;
+  protected final String search;
```

Comment:
Based on the values for search, it seems like this might be better off as an enum class.

Reply:
done